orbitdeterminator/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/kingshark/.virtualenvs/orbit_determinator/bin/python"
```

Comment:
I think we do not need this ;).
please remove.

Reply:
actually, i would need that on my machine, it isn't needed though.

Reply:
You can add the `.vscode` folder to `.gitignore`, so that git does not keep track of those files

Reply:
Unresolving this, since file is still tracked by git... You can un-track this file on git (e.g., `git rm --cached <FILE>`), and then add the following line:

```
.vscode/settings.json 
```

to `.gitignore`

Reply:
@PerezHz shall I just add it to .gitignore as you stated it?

Reply:
@hornig sure, I think it's sufficient to add `orbitdeterminator/.vscode`